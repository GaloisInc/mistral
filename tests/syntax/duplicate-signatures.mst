module Main where
{ f : Int
; f : Char
; f = 10
}
