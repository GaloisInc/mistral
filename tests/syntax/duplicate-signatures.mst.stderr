 
[error] tests/syntax/duplicate-signatures.mst:4,3-4,4
  multiple signatures for binding `f`
    tests/syntax/duplicate-signatures.mst:2,3-2,4
    tests/syntax/duplicate-signatures.mst:3,3-3,4
